docs/BreakingChanges/083 SqlConnection.Open fails on Windows 7 with non-IFS Winsock BSP or LSP present.md

```diff
@@ -0,0 +1,29 @@
+## 83: SqlConnection.Open fails on Windows 7 with non-IFS Winsock BSP or LSP present
```

Comment:
{
  "Id": "83",
  "Title": "SqlConnection.Open fails on Windows 7 with non-IFS Winsock BSP or LSP present",
  "Details": "SqlConneciton.Open and OpenAsync fail in the .NET Framework 4.5 if running on a Windows 7 machine with a non-IFS Winsock BSP or LSP are present on the computer.\n\nTo determine whether a non-IFS BSP or LSP is installed, use the `netsh WinSock Show Catalog` command, and examine every `Winsock Catalog Provider Entry` item that is returned. If the Service Flags value has the `0x20000` bit set, the provider uses IFS handles and will work correctly. If the `0x20000` bit is clear (not set), it is a non-IFS BSP or LSP.",
  "Suggestion": "This bug has been fixed in the .NET Framework 4.5.2, so it can be avoided by upgrading the .NET Framework. Alternatively, it can be avoided by removing any installed non-IFS Winsock LSPs.",
  "Link": "https://connect.microsoft.com/VisualStudio/feedback/details/729323",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Data.SqlClient.SqlConnection.Open",
    "M:System.Data.SqlClient.SqlConnection.OpenAsync",
    "M:System.Data.SqlClient.SqlConnection.OpenAsync(System.Threading.CancellationToken)"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": "4.5.2",
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 2
}

**Parsed APIs:**
- [x] M:System.Data.SqlClient.SqlConnection.Open
- [ ] M:System.Data.SqlClient.SqlConnection.OpenAsync
- [x] M:System.Data.SqlClient.SqlConnection.OpenAsync(System.Threading.CancellationToken)